docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-runtime.adoc

```diff
@@ -99,55 +81,29 @@ The ‘purl’ field identifies the absolute path for the packages.
 // |===
 
 
-[#addressed-issues]
+[#addressed-issues]s
 === Addressed Issues
 [cols="30%a,70%a"]
 |===
 
-//CWP-56818[Doc ticket] CWP-53147 [Engg. Ticket]
-|*Missing Node Count for EKS Clusters in SaaS Environment*
-|Previously, node count was not appearing for EKS clusters on *Manage > Cloud accounts* as a part of *Discovery* report. This is now addressed, and the accurate node count is displayed both on the Prisma Cloud console and in https://pan.dev/prisma-cloud/api/cwpp/get-cloud-discovery/[API Cloud Discovery scan results].
-
-//CWP-56591 [Doc Ticket] CWP-50970 [Eng Ticket]
-//|*Inconsistency with jar packages names*
-//|Fixed an issue where old jar versions were not parsed properly.
+//CWP-56591
+//TODO:Pending blurb approval
+|*Enhanced CVE Detection for Older JAR Versions*
+|Fixed an issue where incorrect parsing of older JAR file versions resulted in missing CVE entries.
 
-//CWP-55859 [Eng Ticket] CWP-56294 [Doc Issue]
-//FIXME: No Doc ticket
-|*`incidentTime` (timestamp) macro for Webhooks*
-|The `incidentTime` macro is available for webhook alerts. The macro shows the time the incident occurred. For example, `Jan 21, 2018 UTC`.
+//CWP-58016
+|*Add ‘risk factor’ to the CSV downloads for Functions and host*
+|Fixed an issue where “risk factor” field was missed on Function and Host CSV results.
```

Comment:
Suggest using the following syntax since we're referring to a UI component.

Fixed an issue where `risk factor` field was missed on Function and Host CSV results.

Reply:
This blurb was removed from the LA as per Eng/PM decision @jenjoe22.